docs/input/docs/configuration.md

```diff
@@ -137,6 +137,16 @@ When using `mode: ContinuousDeployment`, the value specified in
 `continuous-delivery-fallback-tag` will be used as the pre-release tag for
 branches which do not have one specified. Default set to `ci`.
 
+Just to clarify: For a build name without "...-ci-\<buildnumber>" or in other
```

Comment:
Is the backslash in `...-ci-\<buildnumber>` intentional?

```suggestion
Just to clarify: For a build name without `...-ci-\<buildnumber>` or in other
```

Reply:
Now it is obsolete. It is needed in favor of escape the angle bracket. But when surrounded by backticks to represent code it is not needed anymore.

Reply:
So should my suggested change be commited, then?

Reply:
Yea, but I never authored pull request changes, I accidentally created a bunch of unnecessary commits. Sorry. I am not sure. I want to take your change, but then I end with a fresh commit. But their is one change left off, as the escape is not needed anymore. But then I would create another commit to remove that single backslash. So all these commits would be visible when this pull request is merged into develop/master, right? What so ever, I manually added the line into his own commit.